Source/CombatExtended/CombatExtended/SecondaryDamage.cs

```diff
@@ -22,6 +22,7 @@ public DamageInfo GetDinfo(DamageInfo primaryDinfo)
         {
             var dinfo = new DamageInfo(def,
                             amount,
+                            primaryDinfo.ArmorPenetrationInt, //Armor Penetration
```

Comment:
Don't have all the code in my head right now but IIRC this will introduce issues where primary damage can penetrate deeper than secondary and vice versa. Need to review DamageWorker code to make sure secondary damage is applied correctly.

Reply:
After reviewing it looks like the DamageWorker code is going to need some fundamental restructuring, including wrt to how secondary damage is handled.